components/server/src/analytics.ts

```diff
@@ -129,3 +130,31 @@ function stripCookie(cookie: string) {
         return cookie;
     }
 }
+
+export class ContextAwareAnalyticsWriter implements IAnalyticsWriter {
+    constructor(readonly writer: IAnalyticsWriter) {}
+
+    identify(msg: IdentifyMessage): void {}
```

Comment:
I wonder why don't we implement identity and track? I see segment based implementing them, should not we at least delegate?

Reply:
Whoops - should delegate of course :confounded:

Reply:
Done.